addons/website_event/static/src/js/website_event_template_patch.js

```diff
@@ -0,0 +1,23 @@
+/** @odoo-module **/
+
+import publicWidget from "@web/legacy/js/public/public_widget";
+
+publicWidget.registry.websiteEventTemplatePatch = publicWidget.Widget.extend({
+    selector: ".o_wevent_events_list",
+
+    start() {
+        // hack to make event short description editable,
+        // can be removed in master.
+        const eventListEl = this.el.querySelectorAll("small[itemprop='description']");
+        eventListEl.forEach(eventEl => {
+            const id = eventEl.parentElement.querySelector("span[itemprop='name']").getAttribute("data-oe-id");
```

Comment:
A slight linting issues to be resolved here 

```suggestion
        eventListEl.forEach((eventEl) => {
            const id = eventEl.parentElement
                .querySelector("span[itemprop='name']")
                .getAttribute("data-oe-id");
```